code-style/git/README.md

```diff
@@ -322,8 +335,11 @@ Voila! You're done. You've successfully rebased a branch onto the master branch!
 [Draft PRs]: https://github.blog/2019-02-14-introducing-draft-pull-requests/
 [pull_request]: https://help.github.com/articles/using-pull-requests
 [Sparkbox]: http://seesparkbox.com
-[Sprintly]: https://sprint.ly/
-[pull request template]: https://seesparkbox.com/foundry/better_pull_requests_merge_requests_with_templates
+[pull_request_template]: https://seesparkbox.com/foundry/better_pull_requests_merge_requests_with_templates
```

Comment:
It looks like this might be breaking after adding the underscrores.
```suggestion
[pull request template]: https://seesparkbox.com/foundry/better_pull_requests_merge_requests_with_templates
```